test/TestSenseLibrary/app/build.gradle

```diff
@@ -0,0 +1,34 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 21
+    buildToolsVersion "21.1.2"
+
+    defaultConfig {
+        applicationId "nl.sense_os.testsenselibrary"
```

Comment:
@excaliburnan Do we need an app for testing? I guess we could run the test in the sense-android-library library project without creating a separate app.